pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -637,7 +637,7 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 	if !opt.EmptyVols {
 		pvcStr = " and PersistentVolumeClaims "
 	}
-	fmt.Println("We are going to create Kubernetes Deployments, Services" + pvcStr + "for your Dockerized application. \n" +
+	fmt.Println("\nWe are going to create Kubernetes Deployments, Services" + pvcStr + "for your Dockerized application. \n" +
```

Comment:
Remove the `\n`

This may be a good opportunity to change this into a log.Info instead of a Println.